client/src/components/page/Sidebar/index.tsx

```diff
@@ -106,11 +107,13 @@ export const SidebarComponent: React.FC<SidebarProps> = (props: SidebarProps) =>
               </Menu.Item>
             </Menu.SubMenu>
           )}
-          <Menu.Item key="teams">
-            <Link to={URL_TEAMS} className="admin-link">
-              {TEAMS}
-            </Link>
-          </Menu.Item>
+          {ENABLE_TEAMS && (
```

Comment:
Hope we've added the test for this test case as we added for the rest [here](https://github.com/OpenSRP/web/blob/master/client/src/containers/ConnectedSidebar/tests/index.test.tsx#L51)